### PR TITLE
Create extended_euclid_algorithm.cpp

### PR DESCRIPTION
**Pull request by srujangit123**
#### Description of Change
Created a .cpp file in math directory.

Added extended euclid algorithm which is used to find Modular multiplicative inverse, coefficients of a, b in gcd(a, b) = a * x + b * y  etc.

### CODE REVIEW

**Comment by srujangit123** [comment]
Can someone tell why the test is failing?

**Comment by cclauss** [comment]
math/extended_euclid_algorithm.cpp:5:  Lines should be <= 80 characters long  [whitespace/line_length] [2]

File: math/extended_euclid_algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+// Finding coefficients of a and b ie x and y in gcd(a, b) = a * x + b * y
+// d is gcd(a, b)
+// This is also used in finding Modular multiplicative inverse of a number
+// (A * B)%M == 1 Here B is the MMI of A for given M, so extendedEuclid (A, M) gives B
```
Comment (cclauss) [inline math/extended_euclid_algorithm.cpp:5]: math/extended_euclid_algorithm.cpp:5:  Lines should be <= 80 characters long  [whitespace/line_length] [2]

**Comment by srujangit123** [comment]
Modified the file @deadshotsb . Thank you @cclauss

**Comment by cclauss** [comment]
math/extended_euclid_algorithm.cpp:5:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

**Comment by srujangit123** [comment]
Thank you very much for helping @cclauss

**Pull request merged**